fixtures/dmg-lumiere/maxi-mix.json

```diff
@@ -0,0 +1,661 @@
+          "type": "Generic",
+          "comment": "9"
+        }
+      ]
+    },
+    "MemTens": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 24],
+          "type": "Generic",
+          "comment": "0"
+        },
+        {
+          "dmxRange": [25, 49],
+          "type": "Generic",
+          "comment": "1"
+        },
+        {
+          "dmxRange": [50, 74],
+          "type": "Generic",
+          "comment": "2"
+        },
+        {
+          "dmxRange": [75, 99],
+          "type": "Generic",
+          "comment": "3"
+        },
+        {
+          "dmxRange": [100, 124],
+          "type": "Generic",
+          "comment": "4"
+        },
+        {
+          "dmxRange": [125, 149],
+          "type": "Generic",
+          "comment": "5"
+        },
+        {
+          "dmxRange": [150, 174],
+          "type": "Generic",
+          "comment": "6"
+        },
+        {
+          "dmxRange": [175, 199],
+          "type": "Generic",
+          "comment": "7"
+        },
+        {
+          "dmxRange": [200, 224],
+          "type": "Generic",
+          "comment": "8"
+        },
+        {
+          "dmxRange": [225, 255],
+          "type": "Generic",
+          "comment": "9"
+        }
+      ]
+    },
+    "MemSingles": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 24],
+          "type": "Generic",
+          "comment": "0"
+        },
+        {
+          "dmxRange": [25, 49],
+          "type": "Generic",
+          "comment": "1"
+        },
+        {
+          "dmxRange": [50, 74],
+          "type": "Generic",
+          "comment": "2"
+        },
+        {
+          "dmxRange": [75, 99],
+          "type": "Generic",
+          "comment": "3"
+        },
+        {
+          "dmxRange": [100, 124],
+          "type": "Generic",
+          "comment": "4"
+        },
+        {
+          "dmxRange": [125, 149],
+          "type": "Generic",
+          "comment": "5"
+        },
+        {
+          "dmxRange": [150, 174],
+          "type": "Generic",
+          "comment": "6"
+        },
+        {
+          "dmxRange": [175, 199],
+          "type": "Generic",
+          "comment": "7"
+        },
+        {
+          "dmxRange": [200, 224],
+          "type": "Generic",
+          "comment": "8"
+        },
+        {
+          "dmxRange": [225, 255],
+          "type": "Generic",
+          "comment": "9"
+        }
+      ]
+    },
+    "Hue 8bit": {
+      "name": "Hue",
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 37],
+          "type": "Generic",
+          "comment": "0°"
+        },
+        {
+          "dmxRange": [38, 218],
+          "type": "Generic",
+          "comment": "0-360°"
+        },
+        {
+          "dmxRange": [219, 255],
+          "type": "Generic",
+          "comment": "360°"
+        }
+      ]
+    },
+    "Hue 16bit": {
```

Comment:
I think you can drop the `Hue 8bit` channel and just use the `Hue 16bit` channel (renamed to just `Hue`), because the DMX values match exactly:

```
9728 / 256 = 38
```

I.e. when only using the coarse channel, it will downscale the 16-bit values to 8-bit.

Alternatively, you can also define the DMX values as 8-bit numbers, and set `"dmxValueResolution": "8bit"` in the channel. See also the [Fine channel docs](https://github.com/OpenLightingProject/open-fixture-library/blob/schema-12.2.1/docs/fixture-format.md#fine-channels).